### PR TITLE
Show Category

### PR DESCRIPTION
**Pull request by yeunuthongminh**

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    @categories = Category.paginate page: params[:page], per_page: Settings.page
+  end
+
+  def show
+    @words = @category.words
+  end
+
+  private
+  def find_category
+    @category = Category.find_by_id params[:id]
```
Comment (nguyenngocha) [inline app/controllers/categories_controller.rb]: @category == nil?

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+          <%= image_tag "category.jpg", class: "text-center lm-img" %>
+        </div>
+        <p class="text-center lm-cate-name">
+          <%= link_to category.name, category_path(category) %>
+        </p>
+      </div>
+    <% end %>
+  </aside>
+</div>
+
```
Comment (nguyenngocha) [inline app/views/categories/index.html.erb]: bỏ

**Comment by leducphuc** [comment]
ok

File: app/controllers/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    if @category == nil?
```
Comment (hellovietnam93) [inline app/controllers/categories_controller.rb]: unless @category

**Pull request merged**